packages/amplify-e2e-core/src/categories/lambda-function.ts

```diff
@@ -32,6 +32,72 @@ const nodeJSTemplateChoices = [
 
 const pythonTemplateChoices = ['Hello World'];
 
+const additionalPermissions = (chain: ExecutionContext, settings: any) => {
```

Comment:
have you run all of the e2e tests with these changes? This is a pretty substantial change to how the walkthrough works and I feel like there might be some rough edges, especially in how tests from other categories interact with the functions category

Reply:
Discussed offline